bundle-hack/KUSTOMIZE_MIGRATION.md

```diff
@@ -0,0 +1,117 @@
+# Migration from Python to Kustomize/yq
+
+This document explains the new approaches for updating the bundle manifests, replacing the Python-based `update_bundle.sh` script.
+
+## Available Solutions
+
+### 1. yq-based Solution (Recommended)
+
+**File:** `update_bundle_yq.sh`
+
+**Advantages:**
+- Simpler and more maintainable
+- Uses `yq` which is already available (used in `convert.sh`)
```

Comment:
This is not correct, some AI "Hallucination"
YQ is not already available in the build env, but we can add another stage to the dockerfile to make that the case trivially

Reply:
Konflux provide an approved yq image for this purpose